src/core/server/rendering/rendering_service.tsx

```diff
@@ -338,12 +341,14 @@ export class RenderingService {
    * @returns {boolean} indicate if user input title is valid/invalid
    */
   public isTitleValid = (title: string, configName?: string): boolean => {
-    if (!title || title.length > 36) {
+    if (!title) {
+      return false;
+    }
+    if (title.length > 36) {
       this.logger
         .get('branding')
-        .info(
-          configName +
-            ' config is not found or invalid. Title length should be between 1 to 36 characters.'
+        .error(
+          `${configName} config is not found or invalid. Title length should be between 1 to 36 characters.`
```

Comment:
for all these log statements. Can we add "using defaults" or some other message that indicates that there's a fall back?